file_writer_control/JobHandler.py

```diff
@@ -70,10 +70,10 @@ def set_stop_time(self, stop_time: datetime) -> Union[CommandHandler, None]:
         :return: A CommandHandler instance that can be used to monitor the outcome of the attempt to set a new stop time.
         """
         current_status = self.worker_finder.get_job_status(self._job_id)
-        if current_status is None:
-            return None
         return self.worker_finder.try_send_stop_time(
```

Comment:
Is it still possible for the function to return None? We might need to update the return type in line 63.